dev/fattest.simplicity/src/componenttest/topology/utils/ServerFileUtils.java

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.topology.utils;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.topology.impl.LibertyFileManager;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Expands imported files within a Liberty server xml.
+ * The server configs (that include "imports" are assumed to be found in:
+ * publish/servers/<server>/configs/
+ * These config files will have imports located in
+ * publish/shared/config/ of a FAT project
+ *
+ * @author chrisc
+ *
+ */
+public class ServerFileUtils {
+
+    protected static Class<?> thisClass = ServerFileUtils.class;
+    private final String serverConfigDir = "configs";
+    private final String importConfigDir = "config";
```

Comment:
Took me a few minutes to understand your comments and what these actually are. I'd think naming this variable something like `sharedConfigDir` makes more sense since it's the name of the directory pointed to by the `shared.config.dir` environment variable.

Reply:
renamed